src/parser/compile.c

```diff
@@ -746,7 +763,8 @@ pt_add_lock_class (PARSER_CONTEXT * parser, PT_CLASS_LOCKS * lcks, PT_NODE * spe
       return MSGCAT_RUNTIME_OUT_OF_MEMORY;
     }
 
-  sm_downcase_name (spec->info.spec.entity_name->info.name.original, lcks->classes[lcks->num_classes], len + 1);
+  memcpy (lcks->classes[lcks->num_classes], realname, len + 1);
+  lcks->classes[lcks->num_classes][len] = '\0';
```

Comment:
This code can be omitted as it is included when copying the length of len+1 in memcpy().

Reply:
I will remove the below.
```
lcks->classes[lcks->num_classes][len] = '\0';
```